webpack/components/extensions/HostDetails/Tabs/__tests__/errataTab.test.js

```diff
@@ -783,3 +790,418 @@ test('Can filter by severity', async (done) => {
   assertNockRequest(autoSearchScope);
   assertNockRequest(scope2, done); // Pass jest callback to confirm test is done
 });
+
+test('apply button chooses katello agent if enabled', async (done) => {
+  const autocompleteScope = mockForemanAutocomplete(nockInstance, autocompleteUrl);
+  const mockErrata = makeMockErrata({});
+  const options = renderOptions({
+    ...contentFacetAttributes,
+    katelloAgentInstalled: true,
+    katelloAgentEnabled: true,
+  });
+
+  const scope = nockInstance
+    .get(hostErrata)
+    .query(defaultQuery)
+    .reply(200, mockErrata);
+
+  const resolveErrataScope = nockInstance
+    .put(applyByKatelloAgentUrl)
+    .reply(201, mockResolveErrataTask);
+
+  const { getAllByText, getByLabelText, queryByText } = renderWithRedux(
+    <ErrataTab />,
+    options,
+  );
+    // Assert that the errata are now showing on the screen, but wait for them to appear.
+  await patientlyWaitFor(() => expect(getAllByText('Important')[0]).toBeInTheDocument());
+
+  getByLabelText('Select row 0').click();
+  getByLabelText('Select row 1').click();
+
+  const viaAction = queryByText('Apply');
+  expect(viaAction).toBeInTheDocument();
+  viaAction.click();
+
+  assertNockRequest(autocompleteScope);
+  assertNockRequest(resolveErrataScope);
+  assertNockRequest(scope, done);
+});
+
+test('Can bulk apply via katello agent', async (done) => {
+  const autocompleteScope = mockForemanAutocomplete(nockInstance, autocompleteUrl);
+  const mockErrata = makeMockErrata({});
+  const { results } = mockErrata;
+  const options = renderOptions({
+    ...contentFacetAttributes,
+    katelloAgentInstalled: true,
+    katelloAgentEnabled: true,
+  });
+
+  const scope = nockInstance
+    .get(hostErrata)
+    .query(defaultQuery)
+    .reply(200, mockErrata);
+
+  const postBody = ({ bulk_errata_ids: bulkErrataIds }) => {
+    const { included: { ids }, all } = JSON.parse(bulkErrataIds);
+    return isEqual(ids, [results[0].errata_id, results[1].errata_id]) && !all;
```

Comment:
Whenever you use `[0]` and `[1]` array destructuring can make it a bit cleaner
```suggestion
    const [firstResult, secondResult] = results;
    return isEqual(ids, [firstResult.errata_id, secondResult.errata_id]) && !all;
```